.env.example

```diff
@@ -0,0 +1 @@
+SECRET_API_KEY =
```

Comment:
This one is fine. Or if you want to do it this way, it is also fine
`SECRET_API_KEY=your_api_key`